app/src/main/java/com/github/versus/weather/WeatherService.java

```diff
@@ -0,0 +1,95 @@
+package com.github.versus.weather;
+
+import com.github.versus.posts.Location;
+import com.github.versus.posts.Timestamp;
+
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import retrofit2.Call;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public final class WeatherService {
+    private static final String BASE_URL = "https://weather.visualcrossing.com/";
+    private static final String API_KEY= "H4DNTYBMHPAU7EC3D4KC3W7KW";
```

Comment:
You should remove the API_KEY from this file as it should not be visible.

Reply:
It's not a problem for now